docs/configuring-playbook-synapse.md

```diff
@@ -17,6 +17,51 @@ Alternatively, **if there is no pre-defined variable** for a Synapse setting you
 
 - or, if extending the configuration is still not powerful enough for your needs, you can **override the configuration completely** using `matrix_synapse_configuration` (or `matrix_synapse_configuration_yaml`). You can find information about this in [`roles/matrix-synapse/defaults/main.yml`](../roles/matrix-synapse/defaults/main.yml).
 
+## Synapse OpenID Connect
+
+If you plan to rely only on SSO and don't want your users to ever login with password credentials disable this:
+```yaml
+matrix_synapse_password_config_enabled: true
+```
+
+Add this to allow seamless forwarding to element web app and element android app. Without this setting matrix will ask the user if he trusts the app he tries to login.  
+
+```yaml
+matrix_synapse_sso_client_whitelist:
+  - "https://element.{{ matrix_domain }}/"
```

Comment:
This can be `- "{{ matrix_server_fqn_element }}"`.

Should we consider having these as defaults even, instead of asking people to set them.

The `matrix-synapse` role could have an empty list (`matrix_synapse_sso_client_whitelist: []`), while `group_vars/matrix_servers` does something like this:

```yaml
# We always whitelist `element://element` (even if the web client is disabled).
# Perhaps this could be made configurable, if necessary.
matrix_synapse_sso_client_whitelist: |
  {{
    (["{{ matrix_server_fqn_element }}"] if matrix_client_element_enabled else [])
    +
    ['element://element']
  }}
```

I haven't tested this, by the way.